detections/application/splunk_command_and_scripting_interpreter_delete_usage.yml

```diff
@@ -0,0 +1,62 @@
+name: Splunk Command and Scripting Interpreter Delete Usage
+id: 8d3d5d5e-ca43-42be-aa1f-bc64375f6b04
+version: 1
+date: '2022-05-27'
+author: Michael Haag, Splunk
+type: Anomaly
+datamodel:
+- Splunk_Audit
+description: The following analytic identifies the use of the risky command - Delete - that may be utilized in Splunk to delete some or all data queried for. In order to use Delete in Splunk, one must be assigned the role. This is typically not used and should generate an anomaly if it is used.
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) as lastTime from datamodel=Splunk_Audit.Search_Activity where Search_Activity.search IN ("*| delete*") Search_Activity.search_type=adhoc Search_Activity.user!=splunk-system-user
+  by Search_Activity.search Search_Activity.info Search_Activity.total_run_time Search_Activity.user Search_Activity.search_type 
+  | `drop_dm_object_name(Splunk_Audit)` 
+  | `security_content_ctime(firstTime)` 
+  | `security_content_ctime(lastTime)` 
+  | `splunk_command_and_scripting_interpreter_delete_usage_filter`'
+how_to_implement: To successfully implement this search acceleration is recommended against the Search_Activity datamodel that runs against the splunk _audit index. In addition, this analytic requires the Common Information Model App which includes the Splunk Audit Datamodel https://splunkbase.splunk.com/app/1621/.
+known_false_positives: False positives may be present if this command is used as a common practice. Filter as needed.
+references:
+- https://docs.splunk.com/Documentation/Splunk/latest/Security/SPLsafeguards#Commands_that_trigger_the_warning
+tags:
+  analytic_story:
+  - Splunk Vulnerabilities
+  asset_type: Web Server
+  cis20:
+  - CIS 3
+  - CIS 5
+  - CIS 16
+  cve:
+  - CVE-2022-32154
+  confidence: 80
```

Comment:
We're at a confidence of 80 that every time we see a delete command it is malicious? That seems super duper high.